package com.exadel.development.TestOnseo.service.exceptions;

public class DataMappingException extends RuntimeException {
    public DataMappingException(String message, Throwable cause) {
        super(message, cause);
    }
}
